pkg/collector/corechecks/snmp/internal/report/report_metrics.go

```diff
@@ -153,6 +160,8 @@ func (ms *MetricSender) reportColumnMetrics(metricConfig profiledefinition.Metri
 						log.Tracef("unable to tag snmp.%s metric with interface_config data: %s", symbol.Name, err.Error())
 					}
 					tmpTags = append(tmpTags, interfaceCfg.Tags...)
+
+					tmpTags = append(tmpTags, fmt.Sprintf("dd.internal.resource:ndm_interface_user_tags:%s:%s", deviceID, fullIndex))
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Minor suggestion, we could benefit from having a util method for that. Especially as we'll use the same thing for devices

Reply:
https://github.com/DataDog/datadog-agent/pull/27628/commits/ab05a64e749d46dd57b952a50727775860b8ac2c